docs/mindsdb-docs/docs/sql/create/databases.md

```diff
@@ -102,540 +102,1240 @@ The list of databases supported by MindsDB keeps growing. Here are the currently
+      "table_name": " ",                         --- the Airtable table name
+      "api_key": " "                             --- the API key for the Airtable API
+    };
+    ```
+
+=== "Example"
+
+    ```sql
+    CREATE DATABASE airtable_datasource
+    WITH ENGINE='airtable',
+    PARAMETERS={
+      "base_id": "appve10klsda2",
+      "table_name": "my_table",
+      "api_key": "KdJX2Q5km%5b$T$sQYm^gvN"
+    };
+    ```
 
 ### Amazon Redshift
 
-```sql
-CREATE DATABASE amazonredshift_datasource
-WITH ENGINE='amazonredshift',
-PARAMETERS={
-  "user": "amazonredshift",
-  "port": 5439,
-  "password": "amazonredshift",
-  "host": "127.0.0.1",
-  "database": "test"
-};
-```
+=== "Template"
+
+    ```sql
+    CREATE DATABASE amazonredshift_datasource         --- display name for the database
+    WITH ENGINE='amazonredshift',                     --- name of the MindsDB handler
+    PARAMETERS={
+      "host": " ",                                    --- host name or IP address of the Redshift cluster
+      "port": ,                                       --- port used when connecting to the Redshift cluster
+      "database": " ",                                --- database name used when connecting to the Redshift cluster
+      "user": " ",                                    --- user to authenticate with the Redshift cluster
+      "password": " "                                 --- password used to authenticate with the Redshift cluster
+    };
+    ```
+
+=== "Example"
+
+    ```sql
+    CREATE DATABASE amazonredshift_datasource
+    WITH ENGINE='amazonredshift',
+    PARAMETERS={
+      "host": "127.0.0.1",
+      "port": 5439,
+      "database": "test",
+      "user": "amazonredshift",
+      "password": "password"
+    };
+    ```
 
 ### Amazon S3
 
-```sql
-CREATE DATABASE amazons3_datasource
-WITH ENGINE = 's3',
-PARAMETERS = {
-    "aws_access_key_id": " ",
-    "aws_secret_access_key": " ",
-    "region_name": " ",
-    "bucket": " ",
-    "key": " ",
-    "input_serialization": " "
-};
-```
+=== "Template"
+
+    ```sql
+    CREATE DATABASE amazons3_datasource     --- display name for the database
+    WITH ENGINE='s3',                       --- name of the MindsDB handler
+    PARAMETERS={
+      "aws_access_key_id": " ",             --- the AWS access key
+      "aws_secret_access_key": " ",         --- the AWS secret access key
+      "region_name": " ",                   --- the AWS region
+      "bucket": " ",                        --- name of the S3 bucket
+      "key": " ",                           --- key of the object to be queried
+      "input_serialization": " "            --- format of the data to be queried
+    };
+    ```
+
+=== "Example"
+
+    ```sql
+    CREATE DATABASE amazons3_datasource
+    WITH ENGINE='s3',
+    PARAMETERS={
+        "aws_access_key_id": "PCAQ2LJDOSWLNSQKOCPW",
+        "aws_secret_access_key": "U/VjewPlNopsDmmwItl34r2neyC6WhZpUiip57i",
```

Comment:
Are these strings random? If not, then let put here something random

Reply:
I took examples primarily from the handler readme.md files, like this one: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/s3_handler/README.md